setup.py

```diff
@@ -1,4 +1,5 @@
 from setuptools import setup
+import multiprocessing
```

Comment:
It's there because `nose` requires it. Otherwise I get a disgusting error:
```
(wikithingsdb)msilver@malta:~/wikithingsdb$ python setup.py test
running test
Searching for nose>=1.0
Best match: nose 1.3.7
Processing nose-1.3.7-py2.7.egg

Using /afs/csail.mit.edu/u/m/msilver/wikithingsdb/.eggs/nose-1.3.7-py2.7.egg
running egg_info
writing requirements to wikithingsdb.egg-info/requires.txt
writing wikithingsdb.egg-info/PKG-INFO
writing top-level names to wikithingsdb.egg-info/top_level.txt
writing dependency_links to wikithingsdb.egg-info/dependency_links.txt
writing entry points to wikithingsdb.egg-info/entry_points.txt
reading manifest file 'wikithingsdb.egg-info/SOURCES.txt'
writing manifest file 'wikithingsdb.egg-info/SOURCES.txt'
running build_ext
test_articles_of_class (test_fetch.TestArticlesOfCategory) ... ok
test_articles_of_class_error (test_fetch.TestArticlesOfCategory) ... ok
test_articles_of_hypernym (test_fetch.TestArticlesOfCategory) ... ok
test_articles_of_hypernym_error (test_fetch.TestArticlesOfCategory) ... ok
test_articles_of_type (test_fetch.TestArticlesOfCategory) ... ok
test_articles_of_type_error (test_fetch.TestArticlesOfCategory) ... ok
test_classes_of_hypernym (test_fetch.TestArticlesOfCategory) ... ok
test_classes_of_hypernym_error (test_fetch.TestArticlesOfCategory) ... ok
test_classes_of_article (test_fetch.TestCategoriesOfArticle) ... ok
test_classes_of_article_error (test_fetch.TestCategoriesOfArticle) ... ok
test_hypernyms_of_article (test_fetch.TestCategoriesOfArticle) ... ok
test_hypernyms_of_article_error (test_fetch.TestCategoriesOfArticle) ... ok
test_hypernyms_of_class (test_fetch.TestCategoriesOfArticle) ... ok
test_hypernyms_of_class_error (test_fetch.TestCategoriesOfArticle) ... ok
test_types_of_article (test_fetch.TestCategoriesOfArticle) ... ok
test_types_of_article_error (test_fetch.TestCategoriesOfArticle) ... ok
test_add_camelcase (test_fetch.TestStringConversions) ... ok
test_add_camelcase_thing (test_fetch.TestStringConversions) ... ok
test_add_underscores (test_fetch.TestStringConversions) ... ok
test_remove_camelcase (test_fetch.TestStringConversions) ... ok
test_remove_camelcase_thing (test_fetch.TestStringConversions) ... ok
test_remove_underscores (test_fetch.TestStringConversions) ... ok
test_redirects_of_article (test_fetch.TestSymbolsSynonyms) ... ok
test_redirects_of_article_error (test_fetch.TestSymbolsSynonyms) ... ok

----------------------------------------------------------------------
Ran 24 tests in 0.903s

OK
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/usr/lib/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/usr/lib/python2.7/multiprocessing/util.py", line 284, in _exit_function
    info('process shutting down')
TypeError: 'NoneType' object is not callable
Error in sys.exitfunc:
Traceback (most recent call last):
  File "/usr/lib/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/usr/lib/python2.7/multiprocessing/util.py", line 284, in _exit_function
    info('process shutting down')
TypeError: 'NoneType' object is not callable
```